index.js

```diff
@@ -9,39 +8,22 @@ module.exports = function(jadeOptions, locals) {
     var protocol = require('protocol');
     var options = extend({}, jadeOptions || {});
 
-    protocol.interceptBufferProtocol('file', function(request, callback) {
-      var file = request.url.substr(7);
-      var content = null;
+    protocol.interceptProtocol('file', function(request) {
+        if(request) {
+          var file = request.url.substr(8);
```

Comment:
This is not correct! 'file://' is 7 characters. Cutting 8 characters off the `request.url` makes it invalid.